.gitignore

```diff
@@ -2,3 +2,9 @@ target
 logs
 project/project
 .bsp
+
+# VS Code
```

Comment:
Please do not IDE specific there, use `.git/info/exclude`

Reply:
Sorry I had that changed, but it got lost in rebases